include/identity-based/encryption/boneh-franklin/BonehFranklinIdentityBasedEncryption.h

```diff
@@ -49,7 +49,7 @@ CryptidStatus cryptid_ibe_bonehFranklin_extract(AffinePoint* result, const char
  * @param publicParameters the BF-IBE public parameters
  * @return CRYPTID_SUCCESS if everything went right
  */
-CryptidStatus cryptid_ibe_bonehFranklin_encrypt(BonehFranklinIdentityBasedEncryptionCipherText *result, const char *const message, const size_t messageLength,
+CryptidStatus cryptid_ibe_bonehFranklin_encrypt(BonehFranklinIdentityBasedEncryptionCiphertext *result, const char *const message, const size_t messageLength,
```

Comment:
For me, it feels a bit counter-intuitive, that `publicParameters` is the last parameter. Don't you think it would be better as the first input parameter? For some reason, that would feel more natural for me.

Reply:
The `publicParameters` contains mostly information which influences the arithmetic. Those informations are always used as last parameters in the library. I feel like based on this, it is the correct way to place the `publicParameters` to the end of the parameter list.